lib/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -740,6 +740,13 @@ namespace Opm {
             }
             properties.predictionMode = false;
 
+            properties.BHPH = record.getItem( "BHP" ).hasValue(0)
```

Comment:
The `BHPH` and `THPH` elements should be initialized to zero; no need to explicitly assign to 0.0.